### PR TITLE
New re-deployment scripts for CyHy and BOD bastion instances

### PR DESCRIPTION
**Pull request by dav3r**
Nothing groundbreaking here- these are very similar to the re-deployment scripts that we already have for other types of instances.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]
A stellar addition! :stars:

File: terraform/deploy_new_bastion_ami.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# Strip control characters, then look for the text "id" surrounded by
+# space characters, then extract only the ID from that line.
+#
+# The first sed line has been carefully crafted to work with BSD sed.
+bastion_instance_id=$(terraform state show aws_instance.cyhy_bastion | \
+                           sed $'s,\x1b\\[[0-9;]*[[:alpha:]],,g' | \
+                           grep "[[:space:]]id[[:space:]]" | \
+                           sed "s/[[:space:]]*id[[:space:]]*= \"\(.*\)\"/\1/")
+
+# Terminate the existing reporter instance
```
Comment (felddy) [inline terraform/deploy_new_bastion_ami.sh]: Is this comment correct?

File: terraform/deploy_new_bod_bastion_ami.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# Strip control characters, then look for the text "id" surrounded by
+# space characters, then extract only the ID from that line.
+#
+# The first sed line has been carefully crafted to work with BSD sed.
+bod_bastion_instance_id=$(terraform state show aws_instance.bod_bastion | \
+                           sed $'s,\x1b\\[[0-9;]*[[:alpha:]],,g' | \
+                           grep "[[:space:]]id[[:space:]]" | \
+                           sed "s/[[:space:]]*id[[:space:]]*= \"\(.*\)\"/\1/")
+
+# Terminate the existing reporter instance
```
Comment (felddy) [inline terraform/deploy_new_bod_bastion_ami.sh]: Verify this comment.

**Review by felddy — APPROVED** [review]
Approved.  See comment comments.

File: terraform/deploy_new_bastion_ami.sh (hunk 1 above)
Comment (dav3r) [reply terraform/deploy_new_bastion_ami.sh]: Aha, good catch, thanks!  Fixed in https://github.com/cisagov/cyhy_amis/pull/234/commits/16cf850fd7434d9b2ec402a33b23b43ceab859f5.

File: terraform/deploy_new_bod_bastion_ami.sh (hunk 2 above)
Comment (dav3r) [reply terraform/deploy_new_bod_bastion_ami.sh]: Thanks for the sharp 👀 @felddy!  Fixed in https://github.com/cisagov/cyhy_amis/pull/234/commits/16cf850fd7434d9b2ec402a33b23b43ceab859f5.

**Pull request merged**